tools/modules/EnsEMBL/Web/Object/VR.pm

```diff
@@ -107,7 +107,7 @@ sub get_form_details {
     $self->{_form_details} = {
         id => {
           'label'   => 'Variant identifier',
-          'helptip' => 'dbSNP rsIDs or any synonym for a variant present in the Ensembl Variation database',
+          'helptip' => 'Variants present in the Ensembl Variation database that are co-located with input',
```

Comment:
Can we simplify this tooltip? I suggest 'Known co-located variants'. I don't think we get any value from referencing the ensembl variation database

Reply:
I tried to follow this description we have https://www.ensembl.org/info/docs/tools/vep/vep_formats.html#id 
What about `Identifier of any existing co-located variants`?